build/allowedValues.json

```diff
@@ -12,7 +12,8 @@
   ],
   "numberOfDataNodes" : 50,
   "numberOfClientNodes" : 20,
-  "diskSizes" : ["Small", "Medium", "Large", "XLarge", "XXLarge"],
```

Comment:
I did consider this and kept them in to begin, but then thought that keeping them around, whilst it may be helpful for users coming from an older version of the template, was somewhat noisy. 

The template versions are independent of each other, so adhering to SemVer is less important here in my opinion, and passing an older disk size value as an argument will fail fast, with the error message indicating what values are supported.

I'm not totally against keeping them around until the next major (7.0), if you feel strongly about it?

Reply:
Not strongly, happy to get this is in and move along. Agree with the independent version argument. We've added new required args in the past which would be equally breaking semver wise.